docs/en/operations/storing-data.md

```diff
@@ -114,7 +114,11 @@ Example of disk configuration:
 
 ## Using local cache {#using-local-cache}
 
-It is possible to configure local cache over disks in storage configuration starting from version 22.3. For versions 22.3 - 22.7 cache is supported only for `s3` disk type. For versions >= 22.8 cache is supported for any disk type: S3, Azure, Local, Encrypted, etc. Cache uses `LRU` cache policy.
+It is possible to configure local cache over disks in storage configuration starting from version 22.3. 
+For versions 22.3 - 22.7 cache is supported only for `s3` disk type. For versions >= 22.8 cache is supported for any disk type: S3, Azure, Local, Encrypted, etc.
+For versions >= 23.5 cache is supported only for remote disk types: S3, Azure, HDFS.
```

Comment:
But this is not really true, cache is still supported for local disk, it just should have a different disk type "local_blob_storage "(because of https://github.com/ClickHouse/ClickHouse/pull/48791), but will work a bit differently from disk "local". My bad that I did not properly document it (I thought nobody needs this feature).

Reply:
Document it properly when. I don't really know what is local_blob_storage.
I saw experiments with using cache at Ephemeral NVME to speedup EBS. Not sure is it still used or not.